components/ai_chat/core/browser/conversation_driver_unittest.cc

```diff
@@ -811,4 +829,156 @@ TEST_F(ConversationDriverUnitTest, ModifyConversation) {
             "answer2");
 }
 
+class PageContentRefineTest : public ConversationDriverUnitTest,
+                              public testing::WithParamInterface<bool> {
+ public:
+  void SetUp() override {
+    ConversationDriverUnitTest::SetUp();
+    scoped_feature_list_.InitWithFeatureState(features::kPageContentRefine,
```

Comment:
The reason I put it here is because parent class put its scoped feature list [initialization](https://github.com/brave/brave-core/blob/3f54d37b1a0242eeff4d2d225cab0c01a1136d01/components/ai_chat/core/browser/conversation_driver_unittest.cc#L167-L168) in `SetUp`,
If I put mine in constructor, we will hit DCHECK when destructing
```
  // Note: If this DCHECK fires in a test that uses ScopedFeatureList, it is
  // likely caused by nested ScopedFeatureLists being destroyed in a different
  // order than they are initialized.
  if (!was_reset_) {
    DCHECK_EQ(this, global_);
    global_ = nullptr;
  }
```
```
[ RUN      ] PageContentRefineTest.TextEmbedder/PageContentRefine_Disabled
[81207:259:FATAL:field_trial.cc(504)] Check failed: this == global_ (0x6000023c01e0 vs. 0x6000023c00f0)
0   libbase.dylib                       0x000000011657abe2 base::debug::CollectStackTrace(void const**, unsigned long) + 18
1   libbase.dylib                       0x000000011655b89e base::debug::StackTrace::StackTrace() + 62
2   libbase.dylib                       0x00000001164347de logging::LogMessage::Flush() + 190
3   libbase.dylib                       0x00000001164346ad logging::LogMessage::~LogMessage() + 29
4   libbase.dylib                       0x000000011640f57f logging::(anonymous namespace)::DCheckLogMessage::~DCheckLogMessage() + 111
5   libbase.dylib                       0x000000011640f5de logging::(anonymous namespace)::DCheckLogMessage::~DCheckLogMessage() + 14
6   libbase.dylib                       0x000000011640eea3 logging::CheckError::~CheckError() + 35
7   libbase.dylib                       0x000000011640eef9 logging::CheckError::~CheckError() + 9
8   libbase.dylib                       0x0000000116447528 base::FieldTrialList::~FieldTrialList() + 472
9   brave_unit_tests                    0x0000000109204964 base::test::ScopedFeatureList::Reset() + 100
10  brave_unit_tests                    0x000000010920489f base::test::ScopedFeatureList::~ScopedFeatureList() + 15
11  brave_unit_tests                    0x000000010787dd8e ai_chat::PageContentRefineTest_TextEmbedder_Test::~PageContentRefineTest_TextEmbedder_Test() + 46
```

Reply:
I can move parent's to constructor though so we can initialized it in constructor if you want.